src/main/java/com/example/librarymanagement/dto/AuthorDto.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.librarymanagement.dto
+
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.Pattern
+import jakarta.validation.constraints.Size
+
+data class AuthorDto(
+    val id: Long? = null,
+
+    @field:Size(max = 50, message = "First name must contain no more than 50 characters")
+    @field:Pattern(regexp = "[A-Z][a-z]+", message = "First name must start with a capital letter followed by one or more lowercase letters")
```

Comment:
```suggestion
    @field:Pattern(regexp = "[A-Z][a-z]+", message = "First name must start with a capital letter followed by one or more lowercase letters")
```

move regexp to constant